### PR TITLE
Update Overleaf usage instructions

### PR DESCRIPTION
**Pull request by guyrosin**
This template works with XeLaTex version 2018.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ You modify it until it becomes your own actual thesis:
 Your TeX distribution must include the appropriate packages for supporting the use of Hebrew in (Xe)LaTeX. Specifically, the Culmus fonts should be available, as one of them (David CLM) is used by default. In past versions, the template relied on the `babel` package and the `Jerusalem` and other old-style fonts; this is no longer the case: the [`polyglossia`](http://ctan.org/pkg/polyglossia) and the  [`bidi`](http://ctan.org/pkg/bidi) packages are now in use. Still, Hebrew is tricky and some packages have either compatibility issues with `polyglossia`/`bidi`, or must be loaded before `bidi`.
 
 ### Overleaf support
-The package works seamlessly with overleaf by changing the compiler's setting to TexLive 2017:
-To do so, first clone the overleaf templacte and then click the Menu button on the top left corner and select 2017 for the TexLive version.
+This template works seamlessly with Overleaf by changing the compiler to XeLaTeX version 2018.
```
Comment (eyalroz) [inline README.md:30]: Don't you mean TeXLive 2018?

File: README.md [hunk 2]
```diff
@@ -27,8 +27,8 @@ You modify it until it becomes your own actual thesis:
...
+To do so, download this repository as a zip file and upload it to Overleaf as a new project. Then click the Menu button on the top left corner and select XeLaTeX as the compiler and 2018 as the Tex Live version.
```
Comment (eyalroz) [inline README.md:31]: Should I just update the template on Overleaf?

Comment (guyrosin) [reply README.md:31]: Yeah it would easier to use that way :)

File: README.md (hunk 1 above)
Comment (guyrosin) [reply README.md:30]: It's 2 separate requirements, as I wrote in the next line. Please feel free to edit

File: README.md (hunk 2 above)
Comment (eyalroz) [reply README.md:31]: Ok, so... revert this change then?

**Comment by eyalroz** [comment]
Effectively merged this.

**Pull request closed without merging**